editoast/src/models/train_schedule.rs

```diff
@@ -159,10 +167,23 @@ pub struct ResultStops {
     pub duration: f64,
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct FullResultStops {
+    pub time: f64,
+    pub position: f64,
+    pub duration: f64,
```

Comment:
```suggestion
    #[serde(flatten)]
    pub result_stops: ResultStops
```

and also `imply Deref for FullResultStop` which returns `self.result_stops`. Explicit nesting like this highlights the inclusion hierarchy and prevents any future desynchronization.